### PR TITLE
Refactor network configuration

### LINKED ISSUE #2517: Rename port configurations
- `26500` - gateway which is actually the client API
- `26501` - currently client API but actually the transport between gateway and broker
- `26505` - atomix which is actually the clustering/replication etc.

### PR DESCRIPTION
**Pull request by menski**
- remove unused configuration properties and classes
- rename client API to command API
- rename atomix API to internal API
- align standalone gateway internal port to broker internal port (26502)
- update exposed ports in Dockerfile
- rename package `io.zeebe.protocol.clientapi` to `io.zeebe.protocol.commandapi`

closes #2517

### CODE REVIEW

**Review by ChrisKujawa — COMMENTED** [review]
Looks good, just some small things before we can merge this

File: docs/src/introduction/quickstart.md [hunk 1]
```diff
@@ -46,140 +46,10 @@ To start a Zeebe broker use the `broker` or `broker.bat` file located in the
...
-13:14:42.222 [partition-0] [0.0.0.0:26501-zb-actors-0] INFO  io.zeebe.raft - Joined raft in term 0
-13:14:42.280 [zb-stream-processor] [0.0.0.0:26501-zb-actors-1] INFO  io.zeebe.broker.logstreams - Recovering state of partition 0 from snapshot
-13:14:42.851 [zb-stream-processor] [0.0.0.0:26501-zb-actors-1] INFO  io.zeebe.broker.logstreams - Recovered state of partition 0 from snapshot at position -1
-13:14:43.002 [zb-stream-processor] [0.0.0.0:26501-zb-actors-1] INFO  io.zeebe.broker.incident.processor - Start scanning the log for error events.
-13:14:43.003 [zb-stream-processor] [0.0.0.0:26501-zb-actors-1] INFO  io.zeebe.broker.incident.processor - Finished scanning the log for error events.
-
+23:39:13.167 [] [main] INFO  io.zeebe.util.config - Reading configuration for class class io.zeebe.broker.system.configuration.BrokerCfg from file conf/zeebe.cfg.toml
+23:39:13.246 [] [main] INFO  io.zeebe.broker.system - Scheduler configuration: Threads{cpu-bound: 2, io-bound: 2}.
+23:39:13.270 [] [main] INFO  io.zeebe.broker.system - Version: X.Y.Z
+23:39:13.273 [] [main] INFO  io.zeebe.broker.system - Starting broker with configuration {
```
[the remark this answers is not in the document]
Comment (menski) [reply docs/src/introduction/quickstart.md]: Yes this was the intent

**Comment by menski** [comment]
I just pushed the renaming to `interalApi` and `commandApi` plus the removal of the `commandapi` package. After approval I will squash these changes into the correct commits.

**Review by ChrisKujawa — APPROVED** [review]
Cool thanks :+1:

**Pull request merged**